### DISCUSSION TITLE
How do I set up a jet fire

### DISCUSSION THREAD

**Discussion by gdut21122** [firemodels/fds#12678] [General]
Please help me to answer how to set up a jet fire, such as setting up a hydrogen leak jet fire, thank you

**Comment by drjfloyd** [comment]
Read the FDS Technical Reference Guide, User's Guide, and Verification and Validation Guides. All of the input files in the User's Guide and V&V Guides are available to you -- User's and Verifcation in the Examples folder where you installed FDS, and Validation via the <>Code link on this website (or by cloning the repository).  Look at and run example files related to your problem. Once you feel you have an understanding of how FDS works, start making your own input files. Start with simple problems and slowly add complexity. It is far easier to determine errors in a simple file or a file where only one thing has changed from a earlier file than in a file that immeadiately tries to be very complex.

**Comment by mcgratta** [comment]
It appears that you have already created a discussion thread on this issue. I am closing this one.
